src/recensio/plone/browser/templates/review.pt

```diff
@@ -0,0 +1,85 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      lang="en"
+      metal:use-macro="here/main_template/macros/master"
+      xml:lang="en"
+      i18n:domain="recensio"
+>
+
+  <div metal:fill-slot="content-title">
+    <h2 tal:content="structure context/getDecoratedTitle">username: Title</h2>
+  </div>
+
+  <div metal:fill-slot="content-core">
+    <div id="review"
+         tal:define="
+           searchresults context/getSearchresults | nothing;
+         "
+    >
+
+      <p tal:condition="searchresults">
+        ${searchresults}
+      </p>
+
+      <a class="iframe"
+         id="preview-image"
+         href="${here/absolute_url}/@@pageviewer"
+         rel="contactSheet"
+         tal:define="
+           no_pages XXX|nothing|view/get_no_pages;
```

Comment:
It seems to me this tal define should go because no_pages is unused, see:

- https://github.com/syslabcom/recensio.contenttypes/blob/acf6ca3c962bfcf50600739087973de3ba7ad124/recensio/contenttypes/browser/templates/review.pt#L66-L72